source/MaterialXTest/GenGlsl.cpp

```diff
@@ -0,0 +1,99 @@
+#include <MaterialXTest/Catch/catch.hpp>
+
+#include <MaterialXCore/Document.h>
+
+#include <MaterialXFormat/File.h>
+
+#include <MaterialXGenGlsl/GlslShaderGenerator.h>
+#include <MaterialXGenGlsl/GlslSyntax.h>
+
+#include <vector>
```

Comment:
Are these two needed? If Library.h is included by one of the other includes, then they shouldn't be and I think Document.h does just that.

Reply:
Apparently both are already included. Will remove.